### DISCUSSION TITLE
Load model using CPU

### DISCUSSION THREAD

**Discussion by vitotitto** [NVIDIA-NeMo/NeMo#3719] [Q&A] [answered]
Guys, I am trying to load  model `stt_en_conformer_ctc_large = nemo_asr.models.EncDecCTCModelBPE.from_pretrained(model_name='stt_en_conformer_ctc_large', strict=False)` 

and get the RuntimeError that in in the end. Is there a way to use CPU instead? 

`CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 10.76 GiB total capacity; 780.78 MiB already allocated; 19.56 MiB free; 784.00 MiB reserved in total by PyTorch)`

**Comment by titu1994** [comment, marked as answer]
Similar to Pytorch, you can provide map_location="cpu" to the restore_from() or from_pretrained() methods

**Reply by vitotitto** [reply]
Thank you so much!
